spec/controllers/notifications_controller_spec.rb

```diff
@@ -31,6 +31,14 @@
       get :update, "id" => note.id, :set_unread => "true", :format => :json
     end
 
+    it 'marks a notification as unread without timestamping' do
```

Comment:
You should use `TimeCop` here to change the time. Because this spec runs in the same second as the Notification was created, so the time would be the same anyway.

Also this spec is red in travis, because after save it to the database some milliseconds were truncated. So you need to compare only date, hours, minutes and seconds (or only the one you changed with `TimeCop`).